pkg/api/api.go

```diff
@@ -859,6 +860,9 @@ func (p *pushStamperPutter) Put(ctx context.Context, mode storage.ModePut, chs .
 						}
 						return nil
 					}
+					if errors.Is(err, ErrDevNodeNotSupported) {
```

Comment:
This is not required. This error should be returned anyway.

Reply:
It's not returning this error. That is why I have to write it explicitly.